tests/machine.test.js

```diff
@@ -1150,8 +1150,22 @@ describe(OlmMachine.name, () => {
     });
 
     describe("Request missing secrets", () => {
-        async function getHypotheticalKeyQuery(new_user, new_device, bootstrapCrossSigning) {
-            let initialMachine = await OlmMachine.initialize(new_user, new_device);
+        /**
+         *
+         * Creates an hypothetical response to a key query request for an account with a pre-existing device and identity.
+         *
+         * To be used in test when you want to create a setup where there is an existing device
+         * on the account with cross-signing setup.
```

Comment:
Nit:

```suggestion
         * on the account with cross-signing set up.
```

["setup" is a noun. "set up" is the verb]